### ISSUE TITLE
Beta editor: Vim keymap: Ctrl+u and Ctrl+d don't work as expected

### ISSUE THREAD

**Issue by personalizedrefrigerator** [laurent22/joplin#9981]
### Operating system

Linux

### Joplin version

2.14

### Current behaviour

See https://discourse.joplinapp.org/t/codemirror-6-vim-scroll-issues/35864

To reproduce:
1. Enable Vim keybindings in Options > General > Advanced > Keyboard mode.
2. Open a long note and try to scroll with <kbd>ctrl</kbd>-<kbd>u</kbd> and <kbd>ctrl</kbd>-<kbd>d</kbd> while in normal mode.

### Expected behaviour

<kbd>ctrl</kbd>-<kbd>u</kbd> and <kbd>ctrl</kbd>-<kbd>d</kbd> should scroll.

**Comment by personalizedrefrigerator** [comment]
I suspect that this is related to [where we're registering the `vim()` plugin](https://github.com/laurent22/joplin/blob/dd04a39f3b0add299adc0bf26e74c68813414ce3/packages/editor/CodeMirror/configFromSettings.ts#L59). It might be necessary to use [Prec.high](https://codemirror.net/docs/ref/#state.Prec.high) or declare the plugin before the keymap.

**Comment by hulxv** [comment]
I tried to reproduce the issue but it worked as it was expected. Are there more details about this issue? or it was fixed?

**Comment by roman-r-m** [comment]
I suspect you're not using the beta editor, you need to enable it in settings.

**Comment by RadCod3** [comment]
Managed to fix this with `Prec.high` as suggested. Will make a PR when my other one gets merged. 
Can I ask how you figured that out btw? Is there some other extension that consumes maybe part of the keystroke? 🤔 It behaves similar to just clicking u but not exactly.

**Comment by rodolf0** [comment]
Seems this broke once again with the beta editor in 2.14.22. Disabling the beta editor fixes this.

```
Joplin 2.14.22 (prod, linux)

Client ID: b455ea40d72b42878e962e0abe04c9a8
Sync Version: 3
Profile Version: 46
Keychain Supported: No

Revision: e579eb9

Backup: 1.4.0
```

**Comment by personalizedrefrigerator** [comment]
> 2.14.22

Based on the tags for https://github.com/laurent22/joplin/commit/c40682f16fd0ae837f527dcf045a962aa9ab6430, the fix seems to be in Joplin >= 3.0.1, which is still in pre-release. See Joplin's [release cycle page](https://joplinapp.org/help/about/release_cycle) for an estimated release date.
